client/my-sites/plans/jetpack-plans/i7/jetpack-crm-card.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React, { FC } from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { Button } from '@automattic/components';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const JetpackCrmCard: FC = () => {
+	const translate = useTranslate();
+
+	return (
+		<div className="jetpack-crm-card jetpack-product-card" data-e2e-product-slug="free">
+			<header className="jetpack-crm-card__header">
+				<h3 className="jetpack-crm-card__title">{ translate( 'CRM Free' ) }</h3>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Jetpack CRM Free' )` *ES Score: 7*